### DISCUSSION TITLE
Saraga

### DISCUSSION THREAD

**Discussion by kuzey-soydan** [sonsuzus/sonsuzus#515] [Soru Cevap]
Saraga kelimesi, Endonezya dilinde spor tesisi anlamına gelen sarana olahraga kelimesinin kısaltmasıdır. Sarana kelimesinin sara ön eki ve olahraga kelimesinin ga son eki alınarak oluşturulmuştur. İlginç bir şekilde, sarana kelimesinin sa ön eki ve olahraga kelimesinin raga son eki kullanılarak da oluşturulabilir.

İki dize S ve T'nin kısaltması, kısaltmayı iki boş olmayan alt dizeye bölmenin en az iki farklı yolu varsa ilginçtir; böylece ilk alt dize S'nin ön eki ve ikinci alt dize T'nin son eki olur.

Size S ve T olmak üzere iki dize verildi. Minimum uzunlukta S ve T dizelerinin ilginç bir kısaltmasını oluşturmak veya ilginç bir kısaltma oluşturmanın imkansız olup olmadığını belirlemek istiyorsunuz.

Giriş
İlk satır bir S (1≤|S|≤200000) dizesinden oluşur.

İkinci satır T (1≤|T|≤200000) dizesinden oluşur.

Hem S hem de T dizeleri yalnızca küçük İngilizce harflerinden oluşur.

Çıktı
İlginç bir kısaltma oluşturmak mümkün değilse, -1 çıktısı alın.

Aksi takdirde, minimum uzunlukta S ve T dizelerinin ilginç bir kısaltmasını temsil eden tek bir satırda bir dize çıktısı alın. Birden fazla çözüm varsa, bunlardan herhangi birini çıktı olarak alın.

S = input()
T = input()

i = 0
while i < len(S):
    if S[i] in T:
        print(S[i])
        break
    i += 1
else:
    print(-1)

**Comment by sonsuzus** [comment]
Cevabını ayrı bir cevap olarak gönder. Ayrıca kodunu kod bloğu içine almayı unutma. 

Ayrıca kod hatalı. İstenilen işlemi yapmıyor.
